### DISCUSSION TITLE
多个自定义占位符间的分隔符希望可以更改

### DISCUSSION THREAD

**Discussion by eric1ee94** [jxxghp/MoviePilot#3436] [讨论]
### 当前程序版本

1.6.4

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

现在多个自定义占位符的分隔符为@，没有发现可以更改的设置。希望可以像nastool一样可以自定义分隔符。

**Comment by Fzlwhyc** [comment]
我也曾被这个问题困扰过，后来发现自己在后台代码改也很简单！这里写个准保姆级教程，记录学习成果，也给有需要的人。

我用的是群晖。

一、进入SSH并开启root用户登录。_（这里可以细分好几步，网上教程很多这里略）_

二、**确保MP容器正在运行**，复制并粘贴以下命令，检查所有活动中的容器。
`docker ps`

_[PS：在命令行界面，点击鼠标右键就是粘贴，不用Ctrl+V ]_

三、找到MP容器的ID，点击鼠标左键拖动选中此ID。

_[PS：在命令行界面，鼠标选中就是复制操作，不用Ctrl+C ]_

四、进入MP容器。
`docker exec -it 你的MP容器ID /bin/bash`

五、导航到容器内目录app/core/meta/。
`cd app/core/meta/`

_[PS：容器内完整路径是app/app/core/meta/ ]_

六、使用nano编辑器打开指定文件。
`nano customization.py`

七、按Ctrl+_（下划线键），打开行查找功能，输入18（不支持小键盘数字键），回车，光标会自动定位到第18行。

**八**、修改 self.custom_separator = None，把“None值“替换成你想要的分隔符即可。我自己用的分隔符是小数点：
`self.custom_separator = "."`

九、保存并退出：先按Ctrl+x，再按y，再按回车键。

十、大功告成！重启容器生效。
